easy_phi/app.py

```diff
@@ -169,10 +174,14 @@ def post(self):
         used_by = getattr(self.module, 'used_by', None)
         if used_by is not None:
             self.set_status(400)
-            return {'error': "Module is used by {0}. If you need this module, "
+            self.write({'error': "Module is used by {0}. If you need this module, "
                              "you might force unlock it by issuing DELETE "
-                             "request first.".format(used_by)}
+                             "request first.".format(used_by)})
+            return
         setattr(self.module, 'used_by', auth.user_by_token(self.api_token))
+        global ws
+        #Send update to all clients via WS
+        ws.update_lock(hwconf.modules.index(self.module), getattr(self.module, 'used_by', None))
```

Comment:
ModuleHandler has attribute `slot`
may be we should define used_by in abstract module class to get rid of all these getattr/setattr, as lock concept became a thing already